FacebookLoginModule is not abstract and does not override abstract method onActivityResult(Activity,int,int,Intent) in ActivityEventListener

I am getting this error in react native 0.33

:react-native-facebook-login:compileReleaseJavaWithJavac
/Applications/XAMPP/xamppfiles/htdocs/testRN033/node_modules/react-native-facebook-login/android/src/main/java/com/magus/fblogin/FacebookLoginModule.java:37: error: FacebookLoginModule is not abstract and does not override abstract method onActivityResult(Activity,int,int,Intent) in ActivityEventListener
public class FacebookLoginModule extends ReactContextBaseJavaModule implements ActivityEventListener {
       ^
1 error
:react-native-facebook-login:compileReleaseJavaWithJavac FAILED

--- a/android/src/main/java/com/magus/fblogin/FacebookLoginModule.java
+++ b/android/src/main/java/com/magus/fblogin/FacebookLoginModule.java
@@ -345,6 +345,10 @@ public class FacebookLoginModule extends ReactContextBaseJavaModule implements A
         mCallbackManager.onActivityResult(requestCode, resultCode, data);
     }
 
+    public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
+        mCallbackManager.onActivityResult(requestCode, resultCode, data);
+    }
+
     public void onNewIntent(Intent intent) {
 
     }

